### ISSUE TITLE
Add profile for cmder

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#125387]
Setting up a cmder profile is mentioned in https://code.visualstudio.com/docs/editor/integrated-terminal#_can-i-use-cmders-shell-with-the-terminal-on-windows, this should be auto detected. Note that this is a little different to other profiles in that we need to validate not just that the `path` exists, but also `C:\\cmder\\vendor\\bin\\vscode_init.cmd`

**Comment by Tyriar** [comment]
More complicated than I thought, cmder doesn't have an installer so it could be anywhere here are the installation steps:

![image](https://user-images.githubusercontent.com/2193314/121016865-7ee2c000-c751-11eb-9a84-cba0d3f39ef1.png)
https://cmder.net/

**Comment by Tyriar** [comment]
Some users add the folder to the PATH to get it to show up in the start menu

**Comment by Tyriar** [comment]
Asking on cmder repo https://github.com/cmderdev/cmder/issues/2557

**Comment by DRSDavidSoft** [comment]
@Tyriar Sorry to revive this, but [Tabby](https://github.com/Eugeny/tabby) works great by detecting the `cmder_root` env var, in fact [it's encouraged to set this variable](https://github.com/cmderdev/cmder/wiki/Setting-up-Environment-Variables) if not already set.  Also the installers (such as [this one](https://github.com/mikecentola/cmder_inst)) automatically set this environment variable to the installed path:

https://github.com/mikecentola/cmder_inst/blob/586fe12d732e5ed06da3e0392a39a36285e7522a/cmder_inst.nsi#L311-L316

**Comment by Tyriar** [comment]
@DRSDavidSoft I agree we should do this, plus the terminal profiles feature is much more mature now.

**Comment by DRSDavidSoft** [comment]
Hi there @Tyriar, I hope you're doing well!

I believe This issue is related to loading external profiles, I would appreciate it if you could take a look whenever you can.

#164696

**Comment by Tyriar** [comment]
We're not going to you this for now at least because of security concerns; on shared machines/drives the path on question is an attack vector.

**Comment by Tyriar** [comment]
Tracking adding these "unsafe" profiles back via a confirmation step in https://github.com/microsoft/vscode/issues/167721

**Comment by Tyriar** [comment]
Unsafe profiles are coming back in https://github.com/microsoft/vscode/pull/170193, reopening this as it needs a little more functionality to support cmder as it uses cmd
